contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -85,6 +85,16 @@ extern PLtsql_function 	*find_cached_batch(int handle);
 
 extern SPIPlanPtr	prepare_stmt_exec(PLtsql_execstate *estate, PLtsql_function *func, PLtsql_stmt_exec *stmt, bool keepplan);
 
+extern int sp_prepare_count;
+
+bool insert_bulk_keep_nulls = false;
+int insert_bulk_rows_per_batch = DEFAULT_INSERT_BULK_ROWS_PER_BATCH;
+int insert_bulk_kilobytes_per_batch = DEFAULT_INSERT_BULK_PACKET_SIZE;
+
+int prev_insert_bulk_rows_per_batch = DEFAULT_INSERT_BULK_ROWS_PER_BATCH;
```

Comment:
these can be static. Also, this is not ideal to keep global variables to communicate between TSQL and TDS extensions. A parameter based communication would have been much better.

Reply:
Will create a function to communicate the options between TSQL and TDS in the next commit.